app/views/posts/show.html.erb

```diff
@@ -1,2 +1,12 @@
-<h1>Posts#show</h1>
-<p>Test Post</p>
+<%= stylesheet_link_tag "postshow" %>
+
+<div class='open-post-flex'>
+  <div class='open-post'>
+    <h3><%= @post.title %> by <%= @post.author.name %></h3>
+    <p class='amount'>Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></p>
+    <p class='text'><%= @post.text %></p>
+  </div>
+  <% @post.comments.includes(:user).each do |comment| %>
+    <div class='open-comment'><strong><%= comment.user.name %> :</strong> <%= comment.text %></div>
+  <% end %>
```

Comment:
I'd recommend you try to take note of the wireframe for respective views and ensure your views match these wireframes as this is a requirement👍🏼. For example, see the image below for the post comments.

yours
<img width="1440" alt="image" src="https://github.com/DevKaranJ/Blog/assets/64643995/8b298899-2649-402e-b3d8-e2403ab4a9a5">

wireframe
<img width="457" alt="image" src="https://github.com/DevKaranJ/Blog/assets/64643995/ae3ce807-078f-4a8b-852f-00e7391d7a5e">